util/dockerfiles/README.md

```diff
@@ -11,14 +11,16 @@
 
 ## `chapel/chapel:<version>`
 Supported Chapel versions:
-* [`1.24.0`, `latest` (_1.24.0/Dockerfile_)](https://github.com/chapel-lang/chapel/blob/release/1.24/util/dockerfiles/Dockerfile/)
+* [`1.24.1`, `latest` (_1.24.1/Dockerfile_)](https://github.com/chapel-lang/chapel/blob/release/1.24/util/dockerfiles/Dockerfile/)
+* [`1.24.0` (_1.24.0/Dockerfile_)](https://github.com/chapel-lang/chapel/blob/1.24.0/util/dockerfiles/Dockerfile)
 * [`1.23.0` (_1.23.0/Dockerfile_)](https://github.com/chapel-lang/chapel/blob/1.23.0/util/dockerfiles/Dockerfile)
 
 This is the core image for Chapel. It provides the complete Chapel compiler and runtime.  It can be used to compile and run Chapel programs inside the Docker container. On 64-bit Linux hosts, the compiled Chapel program binary can sometimes be executed outside the container (your mileage may vary). Other Chapel-based Docker images can be created from this image.
 
 ## [`chapel/chapel-gasnet:<version>`](https://hub.docker.com/r/chapel/chapel-gasnet/)
 
-* [`1.24.0`, `latest` (_1.24.0/gasnet/Dockerfile_)](https://github.com/chapel-lang/chapel/blob/release/1.24/util/dockerfiles/gasnet/Dockerfile/)
+* [`1.24.1`, `latest` (_1.24.0/gasnet/Dockerfile_)](https://github.com/chapel-lang/chapel/blob/release/1.24/util/dockerfiles/gasnet/Dockerfile/)
+* [`1.24.0` (_1.23.0/gasnet/Dockerfile_)](https://github.com/chapel-lang/chapel/blob/1.23.0/util/dockerfiles/gasnet/Dockerfile)
```

Comment:
I think there's still a mismatch here between 1.24.0 and _1.23.0

Reply:
Also I've just realized the 1.24.1 dockerfile just is the 1.24 dockerfile. So I think we only need the one link?

Reply:
I'm not sure, but if we're dropping 1.24.0 under chapel/chapel-gasnet, I think we should drop it above under chapel/chapel too